File: src/Services/Gateway/Gateway.API/Models/Order.cs
using System;

namespace MRA.Gateway.Models
{
    public class Order
    {
        public Guid MeetingRoomId { get; set; }
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public DateTime Date { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
    }
}

File: src/Clients/Angular.SPA/src/app/models/order.ts
import { Guid } from 'guid-typescript';

export class Order {
    id: Guid;
    meetingRoomId: Guid;
    meetingRoomName: string;
    username: string;
    userId: Guid;
    date: number;
    startTime: string;
    endTime: string;

    constructor(){
        this.id = Guid.create();
        this.meetingRoomName = "";
        this.meetingRoomId =  Guid.create();
        this.username = "";
        this.userId =  Guid.create();
        this.date = Date.now();
        this.startTime = "";
        this.endTime = "";
    }
}